robot_smach_states/src/robot_smach_states/manipulation/point_at.py

```diff
@@ -19,9 +19,8 @@ def __init__(self, robot, arm_designator, point_at_designator, look_at_designato
         """
         Drive the robot back a little and move the designated arm to place the designated item at the designated pose
         :param robot: Robot to execute state with
-        :param placement_pose: Designator that resolves to the pose to place at. E.g. an EmptySpotDesignator
-        :param arm: Designator -> arm to place with, so Arm that holds entity_to_place, e.g. via
-        ArmHoldingEntityDesignator
+        :param arm_designator : arm to place with, so Arm that holds entity_to_place, E.g. ArmHoldingEntityDesignator
```

Comment:
Yep that would be nice. I am not sure it is possible and which sphinx extension would do this.

Reply:
https://stackoverflow.com/questions/7753805/sphinx-automodule-how-to-reference-classes-in-same-module and https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-syntax

Use ```
:class:'.A'
``` but with backticks instead of quotes

Reply:
This PR is to introduce the docs, not to have them fully perfect. That needs to be done in an other PR fixing the docstrings.